Apptainer.def

```diff
@@ -0,0 +1,60 @@
+# Build an Apptainer SIF file containing a working copy of PySR and its prereqs
+Bootstrap: docker
+From: julia:{{ JULIA_VERSION }}-{{ JULIA_BASE_IMAGE }}
+Stage: jl
+
+%arguments
+	JULIA_VERSION=1.10.4
+	JULIA_BASE_IMAGE=bullseye
+
+Bootstrap: docker
+From: python:{{ PYTHON_VERSION }}-{{ BASE_IMAGE }}
+Stage: runtime
+
+%arguments
+	PYTHON_VERSION=3.12
+	BASE_IMAGE=bullseye
+
+%environment
+	# Use the container Julia binary
+	export PATH="/usr/local/julia/bin:$PATH"
+
+	# Create a stacked environment for additional Julia packages
+	export JULIA_DEPOT_PATH="$HOME/.pysr:/pysr/depot:$JULIA_DEPOT_PATH"
+	export JULIA_LOAD_PATH="$HOME/.pysr:/pysr:$JULIA_LOAD_PATH"
+
+%files from jl
+	/usr/local/julia /usr/local/julia
+
+%files
+	./requirements.txt /pysr/requirements.txt
+	./pyproject.toml /pysr/pyproject.toml
+	./setup.py /pysr/setup.py
+	./pysr /pysr/pysr
+
+%post
+	cd /pysr
+
+	export PATH="/usr/local/julia/bin:$PATH"
+
+	# Install IPython and other useful libraries:
+	pip3 install --no-cache-dir ipython matplotlib
+
+	pip3 install --no-cache-dir -r ./requirements.txt
+
+	# Install PySR:
+	pip3 install --no-cache-dir .
+
+	# Put the Julia dependencies in /pysr/depo
+	mkdir /pysr/depot
+
+	export JULIA_DEPOT_PATH="/pysr/depot:$JULIA_DEPOT_PATH"
```

Comment:
```suggestion
	export JULIA_DEPOT_PATH="/pysr/depot"
```
(Or is there a reason to include `JULIA_DEPOT_PATH` here? I would have expected it to be empty)

Reply:
I was just mimicking what they do in the Julia [docs](https://docs.julialang.org/en/v1/manual/environment-variables/#JULIA_DEPOT_PATH) they explain why there.

Reply:
I think those docs are written with a user environment in mind but since this is inside a container we likely want to strip references to external files?

Reply:
I just tested without the trailing $JULIA_DEPOT_PATH. Seems to work fine. I'll take it out.